sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/CallMedia.java

```diff
@@ -125,4 +126,74 @@ public Response<Void> cancelAllMediaOperationsWithResponse(Context context) {
         return callMediaAsync.cancelAllMediaOperationsWithResponseInternal(context).block();
     }
 
+
+    /**
+     * Sends Dtmf tones
+     * @param targetParticipant the target participant
+     * @param tones tones to be sent
+     * @param operationContext operationContext (pass null if not applicable)
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public void sendDtmf(CommunicationIdentifier targetParticipant, List<DtmfTone> tones, String operationContext) {
```

Comment:
operationContext is optional right? If so, please remove it from this method. Optional parameters should only be included in complex method, which is those ones with "WithResponse" suffix.

Reply:
thank you for your suggestion, done!